install/tools/ipa-replica-conncheck

```diff
@@ -594,7 +593,11 @@ def main():
 
                 # Ticket 5812 Always qualify requests for admin
                 user = principal
-                ssh = SshExec(user, options.master)
+                try:
+                    ssh = SshExec(user, options.master)
+                except RuntimeError as e:
+                    root_logger.warning("WARNING: %s, skipping ssh test" % e)
+                    return
```

Comment:
Hi @martbab 
thank you for the comment. I updated the PR to return 0 explicitely.